scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -155,7 +155,7 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
             <input type="hidden" name="case" value="{{case._id}}">
 
             <!--Sample selection-->
-            <div class="row d-flex justify-content-center">
```

Comment:
Hihi, it seems for full layout it could be row - col - d-flex, but this way is more parsimonious!